src/main/java/org/tikv/common/region/RegionManager.java

```diff
@@ -235,10 +235,12 @@ public Pair<TiRegion, TiStore> getRegionStorePairByKey(
         store = getStoreById(peer.getStoreId(), backOffer);
         if (store.isReachable()) {
           // update replica's index
+          logger.info("Store {} is reachable, use it as store", peer.getStoreId());
```

Comment:
```suggestion
          logger.info("Store {} is reachable, select it", peer.getStoreId());
```

Reply:
this log is moved to another pr now. because it is not a part of health check